tools/docker/builder/openwrt/toolchains/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:16.04
+
+RUN dpkg --add-architecture i386
+
+RUN apt-get -y update && apt-get install --no-install-recommends -y sudo git-core \
```

Comment:
1. We prefer to split it to single package per line.
2. No need for sudo - you are root by default in docker
3. Are you sure we need all the tools below?

Reply:
1. I will fix it according to commentary
2. I will fix it according to commentary
3. Yes, I am sure, this Dockerfile we use for building OpenWRT.